src/plugins/vis_type_vislib/public/line_to_expression.ts

```diff
@@ -32,7 +33,7 @@ export const toExpressionAst = async (vis: Vis, params: any) => {
   if (
     params.visLayers == null ||
     Object.keys(params.visLayers).length === 0 ||
-    !isEligibleForVisLayers(vis)
+    !isEligibleForVisLayers(vis, getUISettings())
```

Comment:
Is it required to make this call here? Or is this call here from vislib plugin also causing undefined issues? I was thinking this issue was only happening at external plugin fn calls.

Reply:
Let me test with removing this. I thought this would just be a safer bet mainly.

Reply:
Got it. IMO I'd like to try to keep existing working code in OSD as-is, and only have the workaround in the external plugins to ensure that we have the issue scoped correctly and that it is not also causing issues with other internal OSD plugins.

Reason being, so far it seems that this issue is only caused by some bootstrapping issue when building the compiled application where it is doing the extra installations on the AD/alerting plugins.

Reply:
And please update #4138 if you are seeing this coming from these internal plugins (vislib, visualizations, etc.).

Reply:
It works with removing this, so updated the PR accordingly.